docs/controls/controls.rst

```diff
@@ -0,0 +1,54 @@
+=====================
+Qt Quick Controls 2.0
+=====================
+
+.. sectionauthor:: `e8johan <https://github.com/e8johan>`_
+
+.. github:: ch105
+
+.. note::
+
+    Last Build: |today|
+
+    The source code for this chapter can be found in the `assets folder <../../assets>`_.
+
+This chapter introduces how to use the Qt Quick Controls 2.0 to create a user interface built from standard components such as buttons, labels, sliders and so on. We will look at how various styles can be used to move user interfaces between platforms before diving into custom styling.
+    
+Introduction to Controls
+========================
+
+- What controls exist
```

Comment:
I think this should say "Which controls exist". Or maybe "The list of controls" would be better.

Reply:
This bullet is more of a placeholder for the contents planned. I want to add a set of paragraphs answering this question.